src/plumtree_metadata_manager.erl

```diff
@@ -46,6 +46,9 @@
          graft/1,
          exchange/1]).
 
+%% used by cleanup
+-export([force_delete/1]).
```

Comment:
hey @dergraf This function does not seem to be in use anywhere. Is it kept around for debugging purposes or just dead code?